openlr/paths_connector.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "openlr/graph.hpp"
+#include "openlr/openlr_model.hpp"
+#include "openlr/stats.hpp"
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+namespace openlr
+{
+class PathsConnector
+{
+public:
+  PathsConnector(double const pathLengthTolerance, Graph const & graph, v2::Stats & stat)
```

Comment:
Please, move ctor definition to the cpp part.

Reply:
Done.